src/main/java/com/commercetools/payment/handler/CommercetoolsLookUpPaymentsController.java

```diff
@@ -0,0 +1,32 @@
+package com.commercetools.payment.handler;
+
+import com.commercetools.pspadapter.paymentHandler.PaymentHandlerProvider;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.propertyeditors.StringTrimmerEditor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.annotation.Nonnull;
+import java.util.concurrent.CompletionStage;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@RestController
+public class CommercetoolsLookUpPaymentsController extends BaseCommercetoolsPaymentsHandleController {
+
+    public CommercetoolsLookUpPaymentsController(@Nonnull StringTrimmerEditor stringTrimmerEditor,
+                                                 @Nonnull PaymentHandlerProvider paymentHandlerProvider) {
+        super(stringTrimmerEditor, paymentHandlerProvider);
+    }
+
+    @GetMapping(
+            value = "/{tenantName}/commercetools/look-up/payments/{paypalPaymentId}",
```

Comment:
You are right that it's not the best name ever, but it's used across Paypal - in the docs, the PP guy also always use `look up`. And as this is exposed to the frontend, I thought that we should use similar name so that when they read the docs, they can immediately know what the endpoint will do.

Reply:
but this name is not the same as usual REST name (even though it is regular REST request) and not the same as the Paypal Plus REST API name. Since we are developers and we are doing it for developers - let's stick with conventional names, even if some documentation call it different.

Reply:
What about URL like this: `/{tenantName}/paypalplus/payment/{paypalPaymentId}`? Since user needs to input paypal payment id, I removed `commercetools` from path and put `paypalplus` instead.

Reply:
then we need to update all the controllers URL.
@butenkor what do you think: could we remove `commercetools` from the controllers urls or we should somehow to adjust this URL so we explicitly mention this is Paypal Plus payment, not commercetools?

The urls now look like:
`/{tenantName}/commercetools/create/payments/{ctpPaymentId}`
`/{tenantName}/commercetools/execute/payments`
`/{tenantName}/commercetools/patch/payments/{ctpPaymentId}`
`/{tenantName}/paypalplus/notification` (for notifications listening)

Reply:
however, for other controllers we don't use `paypalPaymentId`, that's why I suggest to leave it as it's now, so FE knows that they need to supply `ctpPaymentId`.

Reply:
https://github.com/commercetools/commercetools-paypal-plus-integration/pull/74/commits/93f18e717b37e352104d284a2ad1b51c7c12e951